Makefile.am

```diff
@@ -32,6 +32,14 @@ service_DATA = \
 	search-provider/com.endlessm.EknServices2.SearchProviderV2.service \
 	$(NULL)
 
+search-provider/eks-article-metadata-query-provider-dbus.h search-provider/eks-article-metadata-query-provider-dbus.c: search-provider/eks-article-metadata-query-provider-dbus.xml Makefile.am
```

Comment:
This is called both "article metadata provider" and "article metadata query provider". It would be best to pick only one, and my preference would be the even shorter "metadata provider" since it provides metadata for things that are not articles as well.

Reply:
I'll change it to "metadata provider"